package.json

```diff
@@ -20,11 +20,11 @@
     "mini-css-extract-plugin": "^0.9.0",
     "node-sass": "^6.0.1",
     "optimize-css-assets-webpack-plugin": "^5.0.3",
-    "pa11y-ci-reporter-html": "^2.1.2",
     "pa11y-ci": "^2.4.0",
+    "pa11y-ci-reporter-html": "^2.1.2",
     "sass-loader": "^10.1.1",
     "style-loader": "^1.1.3",
-    "webpack": "^4.41.6",
+    "webpack": "^4.46.0",
```

Comment:
Can you revert the changes to `package.json` as well as the new lockfiles? We should scope this PR to only adding the CSS classes and tackle upgrading dependencies in a different PR.

See [the new contributing guidelines](https://pydata-sphinx-theme.readthedocs.io/en/latest/contributing.html#build-the-documentation) for how to test the documentation without upgrading the packages (you should just need to run `nox -s docs` if you are on the master branch)

Reply:
fixed in 2efe330ed13e7ed0566d2518f628744578043151